.github/workflows/build-images-and-create-deployment.yml

```diff
@@ -16,11 +16,6 @@ env:
   DOCKER_REPO: ${{ secrets.ECR_REPO }}/
   GITHUB_BRANCH: ${{ github.event.client_payload.ref || github.ref }}
 
-# add a concurrency group to prevent multiple builds from running at the same time
```

Comment:
@atolopko-czi @tihuan - I am removing `concurrency` so as to complicate things. I was confused by dependency between _"staging  happy stack deploy" and "push tests" on main_ which was caused due to the concurrency groups (see image below)
<img width="1707" alt="Screenshot 2023-10-18 at 11 25 19 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/5581819/57025c53-0646-49c1-88ba-6ec91b9c927a">



I will address concurrency in the next PR.